tc/sgx/common/crypto/sig_public_key.cpp

```diff
@@ -296,22 +296,20 @@ std::string pcrypto::sig::PublicKey::SerializeXYToHex() const {
 }  // SerializeXYToHex
 
 // XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
-// Verifies SHA256 ECDSA signature of message
+// Verifies ECDSA signature of message
 // input signature ByteArray contains raw binary data
```

Comment:
Can we change raw binary data to hash of binary data?

Reply:
As far this function is concerned, it treats input as raw binary data. It need not be aware of the source of data (hashed or not).

Reply:
ok